script/hypercube/hypercube.sh

```diff
@@ -352,14 +345,14 @@ function deb_setlocales_and_tz() {
 function ynh_postinstall() {
   logfile ${FUNCNAME[0]}
 
-  yunohost tools postinstall -d "${settings[yunohost,domain]}" -p "${settings[yunohost,password]}" --debug |& logfilter
+  yunohost tools postinstall -d "${settings[yunohost,domain]}" -p "${settings[yunohost,password]}" &>> $log_file
 }
 
 function ynh_addappslist() {
   logfile ${FUNCNAME[0]}
 
-  yunohost app fetchlist -n community -u https://app.yunohost.org/community.json --debug &>> $log_file
-  yunohost app fetchlist --debug &>> $log_file
+  yunohost app fetchlist -n community -u https://app.yunohost.org/community.json &>> $log_file
+  yunohost app fetchlist &>> $log_file
```

Comment:
We shouldn't do this now, because vpnclient_ynh and hotsport are directly manage in the official list of yunohost.